### PR TITLE
Correct data refresh status message next steps

### LINKED ISSUE #887: Data refresh logging steps are out of order
## Description

The log messages we get from the data refresh now are no longer in the correct order. Here are some examples:

**Previous order**:
Screenshot_2022-08-17_13-53-00

**Current order**:
Screenshot_2022-08-17_13-53-21

We should alter the slack messages so that they appropriately convey the current and next steps. The next step is included because the refresh process can take several days, so it can be useful to know which step the refresh is on without waiting until said step completes.

## Resolution

- [ ] 🙋 I would be interested in resolving this bug.

### PR DESCRIPTION
**Pull request by AetherUnbound**
## Fixes

Fixes #887 by @AetherUnbound

## Description

This PR builds off of the changes in #707 and corrects some of the messaging around next steps within the data refresh update steps.

I've also added a new `slack::status` method which will prepend the model name to the beginning of the slack message, which should hopefully make it easier to be more consistent around the messaging. Admittedly "status" is a pretty generic name, if folks think `data_refresh_status` or something else would be better, happy to rename it! 

I've added a just recipe for the index deletion step too, which is in-line with the other recipes we have around indexing.

image

## Testing Instructions

The best way to run this would be to add a `SLACK_WEBHOOK` URL to the `ingestion_server/env.docker` file, then run the following:

1. `just recreate` - this will run the audio and image data refreshes, and display messages associated with creating index aliases towards the end.
1. `just ingest-upstream image new-test` - this will start the process for creating a new index called `new-test`
1. `just promote image new-test` - run this once the previous step gets to "Indices & constraints applied | _Next: table promotion_" step
1. `just delete image` - run this once the previous step gets to "Migrated alias image from index image-init to index image-new-test"

This will emulate the entire process of a data refresh, and you should see all the messages we're likely to see during a normal data refresh process!

## Checklist

- [x] My pull request has a descriptive title (not a vague title like `Update index.md`).
- [x] My pull request targets the *default* branch of the repository (`main`) or a parent feature branch.
- [x] My commit messages follow [best practices][best_practices].
- [x] My code follows the established code style of the repository.
- [ ] I added or updated tests for the changes I made (if applicable).
- [x] I added or updated documentation (if applicable).
- [x] I tried running the project locally and verified that there are no visible errors.

[best_practices]:https://git-scm.com/book/en/v2/Distributed-Git-Contributing-to-a-Project#_commit_guidelines

## Developer Certificate of Origin

<details>
<summary>Developer Certificate of Origin</summary>

```
Developer Certificate of Origin
Version 1.1

Copyright (C) 2004, 2006 The Linux Foundation and its contributors.
1 Letterman Drive
Suite D4700
San Francisco, CA, 94129

Everyone is permitted to copy and distribute verbatim copies of this
license document, but changing it is not allowed.


Developer's Certificate of Origin 1.1

By making a contribution to this project, I certify that:

(a) The contribution was created in whole or in part by me and I
    have the right to submit it under the open source license
    indicated in the file; or

(b) The contribution is based upon previous work that, to the best
    of my knowledge, is covered under an appropriate open source
    license and I have the right under that license to submit that
    work with modifications, whether created in whole or in part
    by me, under the same open source license (unless I am
    permitted to submit under a different license), as indicated
    in the file; or

(c) The contribution was provided directly to me by some other
    person who certified (a), (b) or (c) and I have not modified
    it.

(d) I understand and agree that this project and the contribution
    are public and that a record of the contribution (including all
    personal information I submit with it, including my sign-off) is
    maintained indefinitely and may be redistributed consistent with
    this project or the open source license(s) involved.
```

</details>

### CODE REVIEW

File: ingestion_server/ingestion_server/slack.py [hunk 1]
```diff
@@ -68,3 +68,13 @@ def info(text: str, summary: str = None) -> None:
 
 def error(text: str, summary: str = None) -> None:
     _message(text, summary, level=Level.ERROR)
+
+
+def status(model: str, text: str) -> None:
```
Comment (sarayourfriend) [inline ingestion_server/ingestion_server/slack.py:73]: Given this is just for the ingestion server module I think `status` is a fine name :slightly_smiling_face:

**Review by sarayourfriend — APPROVED** [review]
LGTM. Really nice clean up and consistency improvements to the messages :rocket:

**Review by dhruvkb — APPROVED** [review]
The changes look good. Is this PR related to #717?

**Review by stacimc — APPROVED** [review]
Looks great to me, thank you for not letting this slip through the cracks! I like the addition of `slack::status` very much :)

**Comment by AetherUnbound** [comment]
> Is this PR related to #717?

I suppose it is in that it touches the Slack logging aspects! I believe that ticket is still valid though and could be worked on. It wouldn't be too bad if the log message also included the `media_type` in that implementation.

**Pull request merged**